File: app/src/main/java/vergecurrency/vergewallet/service/model/wallet/BNBalance.java
package vergecurrency.vergewallet.service.model.wallet;

import com.google.gson.Gson;

public class BNBalance {

	private long confirmed;
	private long unconfirmed;
	private long balance;

	public static BNBalance decode(String JSON) {
		return new Gson().fromJson(JSON, BNBalance.class);
	}

	public long getConfirmed() {
		return confirmed;
	}

	public void setConfirmed(long confirmed) {
		this.confirmed = confirmed;
	}

	public long getUnconfirmed() {
		return unconfirmed;
	}

	public void setUnconfirmed(long unconfirmed) {
		this.unconfirmed = unconfirmed;
	}

	public long getBalance() {
		return balance;
	}

	public void setBalance(long balance) {
		this.balance = balance;
	}
}

File: app/src/main/kotlin/vergecurrency/vergewallet/service/model/wallet/BNBalance.kt
package vergecurrency.vergewallet.service.model.wallet

import com.google.gson.Gson

class BNBalance {

    var confirmed: Long = 0
    var unconfirmed: Long = 0
    var balance: Long = 0

    companion object {

        fun decode(JSON: String): BNBalance {
            return Gson().fromJson(JSON, BNBalance::class.java)
        }
    }
}
